src/MOI/MOI_wrapper.jl

```diff
@@ -51,6 +51,12 @@ const SCALAR_SETS = Union{
     MOI.Interval{Float64},
 }
 
+const XPRS_ATTRIBUTES = Dict{String, Any}(
+    replace(string(name), "XPRS_"=>"") => getfield(Lib, name)
+        for name in names(Lib; all = true)
+        if startswith(string(name), "XPRS_") && all(isuppercase(c) for c in string(name) if c != '_')
+)
+
```

Comment:
I had already implmented similar stuff in:
```
    const parameter_name_to_index = Dict{String, Int}(
        string(name) => getfield(Lib, name)
        for name in names(Lib; all = true)
        if startswith(string(name), "XPRS_") && typeof(getfield(Lib, name)) == Int
    )
```

Reply:
Fixed in c2f5ffbeda74470da463301f002da1c95704a628. 

I chose to remove `parameter_name_to_index` and use `XPRS_ATTRIBUTES` everywhere.

1) using `typeof(getfield(Lib, name)) == Int` does not get attributes that are not integers e.g.

```
const XPRS_OBJ_MINIMIZE = 1.0
const XPRS_OBJ_MAXIMIZE = -1.0
...
const XPRS_ISUSERSOLUTION = 0x01
const XPRS_ISREPROCESSEDUSERSOLUTION = 0x02
```

2) I think we don't want to require that the user type `XPRS_$attribute` every time. I don't feel strongly about this though.

Thoughts?

Reply:
agree with 1 and 2